patterns/2-structured/praise-participants.md

```diff
@@ -80,5 +83,5 @@ Overdoing it may feel insincere and mechanical and defeat your purpose in reachi
 
 ## Acknowledgments
 
-* [Todd Lisonbee](https://github.com/tlisonbee) for encouraging to "keep it real".
-* [Isabel Drost-Fromm](https://github.com/MaineC) for [this extra explanation](https://youtu.be/h3MPewsk5PU?t=357) of a "qualified" thank you.
+* [Todd Lisonbee](https://github.com/tlisonbee) for encouraging to "keep it real."
+* [Isabel Drost-Fromm](https://github.com/MaineC) for [this extra explanation](https://youtu.be/h3MPewsk5PU?t=357) of the importances of providing specific examples of how the contribution has positively impacted the project.
```

Comment:
```suggestion
* [Isabel Drost-Fromm](https://github.com/MaineC) for [this extra explanation](https://youtu.be/h3MPewsk5PU?t=357) of the importances of providing specific examples of how the contribution has positively impacted the project. _Aside_: Some people have called this a "qualified thank you", however that may be a German expression that does not translate that well into English.
```

Reply:
@semioticrobotic how does this sound to you? Feel free to fix if the are mistakes in here.

Reply:
This looks great. Thanks!